silx/gui/plot/items/image.py

```diff
@@ -459,3 +458,73 @@ class MaskImageData(ImageData):
     internal silx widgets.
     """
     pass
+
+
+class ImageStack(ImageData):
+    """Item to store a stack of images and to show it in the plot as one
+    of the images of the stack.
+
+    The stack is a 3D array ordered this way: `frame id, x, y`.
```

Comment:
I think there is a typo here: `frame id, y, x`

Reply:
Thanks a lot, i was not sure about that (and did not checked). As it is images inside, but in fact y,x is the normal order for image too.